<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningDialog</class>
 <widget class="QDialog" name="WarningDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pH Calibration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="userInfo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSerif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cont">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
